Oscil.h

```diff
@@ -150,6 +151,19 @@ class Oscil
 	}
 
 
+	/** Returns the next sample given a phase modulation value.
+	@param phmod_proportion a phase modulation value given as a proportion of the wave. The
+	phmod_proportion parameter is a SFixMath<15,16> fixed-point number where the fractional part represents almost -1 to almost 1, modulating the phase by one whole table length in
+	each direction.
+	@return a sample from the table.
+	*/
+	inline
+	int8_t phMod(SFixMath<15,16> phmod_proportion)
```

Comment:
Wouldn't this be much more powerful as a template function taking *any* SFixMath?

Reply:
Indeed, like the way I did for `setFreq()` you mean? Will do.